python/README.md

```diff
@@ -0,0 +1,62 @@
+# python
+
+A Wing library for integrating with Python code.
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+```sh
+npm i @winglibs/python
+```
+
+## Usage
+
+```js
+bring python;
+
+let func = new python.Function(
+  path: "./test-assets",
+  handler: "main.handler",
+);
+
+test "invokes the function" {
+  let res = func.invoke();
+}
+```
+
+It is also possible to interact with Wing resources through the python code
+
+```js
+// main.w
+let bucket = new cloud.Bucket();
+let func = new python.Function(
```

Comment:
How would this work with something like e.g. `cloud.Api`?

Ideally this would have been a `python.Inflight` instead of `python.Function` and then maybe I could do:

```js
let api = new cloud.Api();
api.post("/users", python.Inflight(...));
```

Not sure how this would work but that would be my ideal DX, no?

Or event `python.inflight()` (all lowercase)

Reply:
Yes I understand the problem...

Generics...

Can we do something like this?

```js
python.Api.inflight()
python.Function.inflight()
```